### ISSUE TITLE
Add option for field blocks and display mode - author & date fields

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#3929]
**Description of the need**
We have added a number of great display suites type features. 
https://github.com/backdrop/backdrop-issues/issues/2084
https://github.com/backdrop/backdrop-issues/issues/1100

But, something that seems to be missing is the ability to add date and author information to display modes OR to access field blocks for this information. 

I was just experimenting with how much a site builder can modify their page with Backdrop CMS core and the inability to use existing features to move author and date information feels like an obvious omission. 

**Proposed solution**

There would be options to arrange the author / date field on the display mode. 
There would be a field block for author and/or date submitted. 

**Alternatives that have been considered**
Describe alternatives you've considered. Have you seen this work in a better way in another CMS?

I have yet to look for contrib solutions. 

**Additional information**
Add any other information or screenshots that would help.

![Post__Default_display___Demo](https://user-images.githubusercontent.com/3144571/61193910-da9a9080-a683-11e9-888d-e499afa2fe8c.png)

These are the field blocks currently available to users for the Post content type. 

![Posts___Demo](https://user-images.githubusercontent.com/3144571/61193949-0a499880-a684-11e9-9181-5c17c4d43aca.png)

**Comment by BWPanda** [comment]
@stpaultim Is this a duplicate of, or just related to, #2379?

**Comment by stpaultim** [comment]
I think at most this is related to #2379. #2379 is about changing the format of the 'submitted by' field, this is about placement of the field through display modes and field blocks.

**Comment by BWPanda** [comment]
We did discuss there about creating a pseudo field so that this could be displayed relative to other fields (and configured per display mode), but since that's currently blocked by https://github.com/backdrop/backdrop-issues/issues/3859 then yeah, we're just focused on adding better configuration to core ATM.

**Comment by kswan** [comment]
I am struggling with this issue when using a layout to arrange fields on a node display. The only alternative I have seen is using a Views block, but that feels like a heavy solution.

**Comment by argiepiano** [comment]
@kswan what is the struggle? Adding author and date "fields" to the layout? This is because they are not Field API fields,  they are technically properties, and there is no Block handler to add  properties. But it should be fairly easy to write one. Take a look at implementations of hook_block_info  (e.g. field_block_info).

**Comment by kswan** [comment]
@argiepiano, you are correct. I want to display the author and date line when I use a layout to arrange the fields for a node display. This seems to be a gap in the functionality of layouts. 
From a technical perspective, the difference between properties and fields makes sense as the reason the author and date are not available. But from a functionality/site builder perspective, the author and date line should be available.

**Comment by argiepiano** [comment]
> This seems to be a gap in the functionality of layouts.

Agreed. This should be added to core. 

In the meantime, if helpful, I just created a contrib module that allows you to display node properties like date, author and title in a block. This could hopefully be a first step toward adding this functionality in core.

https://github.com/backdrop-contrib/node_property_block

If you want to give it a try I'll appreciate feedback.

**Comment by indigoxela** [comment]
Providing the properties (author / created date...) as block seems a useful feature in core. Until now we had / have to use workarounds like views blocks.

After some years the issue description might need a little update - @stpaultim what do you think?

**Comment by kswan** [comment]
I think Token block gives the flexibility and functionality that is needed.

Node property block is limited to a few properties and doesn't include the format I am looking for. One option that would make this more useful would be to add the Author/date information defined by the Content Type (but since this is a token string, Token block covers this). 
![image](https://user-images.githubusercontent.com/154391/193056901-f826398e-a39d-448c-b3e0-078e5df93625.png)

@argiepiano Thank you! This is incredibly helpful.

**Comment by argiepiano** [comment]
> One option that would make this more useful would be to add the Author/date information defined by the Content Type

Good idea. I'll add that in the next release of Node Property Block. Thanks for testing, @kswan.
